ui/components/src/Table/TableBody.tsx

```diff
@@ -0,0 +1,21 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { TableBody as MuiTableBody, TableBodyProps as MuiTableBodyProps } from '@mui/material';
+import { forwardRef } from 'react';
+
+type TableBodyProps = MuiTableBodyProps;
+
```

Comment:
A few of these components are functionally just empty wrappers around MUI components because I wasn't sure what I would need to customize when I started. I like the idea of keeping them because they're not hurting anything and it makes it super-obvious where everything is if you need to make changes.